docs/Configuration/yaml-files.md

```diff
@@ -326,20 +341,40 @@ software:
    - app_store_id: '1091189122'
 ```
 
-#### packages
+##### packages
 
 - `url` specifies the URL at which the software is located. Fleet will download the software and upload it to S3 (default: `""`).
 - `install_script.path` specifies the command Fleet will run on hosts to install software. The [default script](https://github.com/fleetdm/fleet/tree/main/pkg/file/scripts) is dependent on the software type (i.e. .pkg).
 - `pre_install_query.path` is the osquery query Fleet runs before installing the software. Software will be installed only if the [query returns results](https://fleetdm.com/tables/account_policy_data) (default: `""`).
 - `post_install_script.path` is the script Fleet will run on hosts after intalling software (default: `""`).
 - `self_service` specifies whether or not end users can install from **Fleet Desktop > Self-service**.
 
-#### app_store_apps
+##### app_store_apps
 
 - `app_store_id` is the ID of the Apple App Store app. You can find this at the end of the app's App Store URL. For example, "Bear - Markdown Notes" URL is "https://apps.apple.com/us/app/bear-markdown-notes/id1016366447" and the `app_store_id` is `1016366447`.
 
 > Make sure to include only the ID itself, and not the `id` prefix shown in the URL. The ID must be wrapped in quotes as shown in the example so that it is processed as a string. 
 
+##### Separate file
+
+`lib/software/tailscale.exe.software.yml`:
```

Comment:
```suggestion
`lib/software-name.package.yml`:
```

I think let's be consistent and use a generic example name like we do for queries [here](https://fleetdm.com/docs/configuration/yaml-files#separate-file2):

![Screenshot 2024-09-23 at 9 57 03 AM](https://github.com/user-attachments/assets/63f1fb0a-da1c-4f34-9c7c-7a8dd80f0a7c)

In practice I think this will look like `lib/windows-tailscale.package.yaml`

I also like `.package` b/c it's explicit that it's a package and not an app store app.